### DISCUSSION TITLE
Edit Series

### DISCUSSION THREAD

**Discussion by cmgorton** [forem/forem#15673] [Icebox]
Original requests by [Eduardo Julião](https://forem.dev/eduardojuliao/edit-series-4fi) and [Uzair Ali](https://forem.dev/uzairali10/order-blogs-in-a-series-2llj) on forem.dev

Users would like the ability to do the following after a series is created:
- Reorder posts in a series
- change the name of a series

Example of a series:
![Series on forem.dev](https://user-images.githubusercontent.com/21039864/144619990-9ed5c471-7f6b-4a1a-8e7c-c6fc79b8aa0e.png)

**Comment by citizen428** [comment]
Just for our internal discussion: series is a topic that probably could use some love in general. Right now we use the term "Series" in user-facing places whereas the model is called `Collection` (though we also use the term `series` in several places in the codebase. In short, this may need to spend some more time with product so we can fully flesh this out.

**Comment by cmgorton** [comment]
Similar discussion from @tvvignesh https://github.com/forem/forem/discussions/15391#discussion-3688846

**Comment by JennieOcken** [comment]
There are a lot of requests around making our series functionality more robust.  I want to group them all together here and say we will take them as a set at some point in the future.

[Write description](https://github.com/forem/forem/discussions/15474)
[Edit](https://github.com/forem/forem/discussions/15673)
[Delete](https://github.com/forem/forem/discussions/16434)
[Embed](https://github.com/forem/forem/discussions/16805)
[Multiple Authors](https://github.com/forem/forem/discussions/15320)
[Reorder](https://github.com/forem/forem/discussions/15391)

**Reply by bhavik009** [reply]
Any update here??

**Comment by michael-tharrington** [comment]
Just upvoted this one! I assisted a community member earlier today with reordering posts in their series and it'd be great to give folks the ability to do this.

**Comment by MortalHappiness** [comment]
+1 for this. Maybe the reordering feature can be implemented like the [Taxonomy weight in Hugo](https://gohugo.io/content-management/taxonomies/#example-taxonomic-weight) via the frontmatter?
